README.md

```diff
@@ -23,6 +23,24 @@ Options:
     -h | help print this help text
 ```
 
+Use Case
+---
+
+For your open source projects, you could add the following to your Github Issue template, replacing `my-module` with the name of your npm module:
```

Comment:
How would you feel about moving this under the initial `trouble---` section? I think it's useful enough to have up front (since it was the first question you asked me lol)

Also we could link out to the only "in the wild" use case [here](https://github.com/NickTomlin/protractor-flake/commit/140f20ba7dbab1c6714ef635051bbab9776fc1ba#diff-faa36bc26a21ed93c8de974753b71507R12)